clarity/src/vm/docs/mod.rs

```diff
@@ -814,12 +814,12 @@ pub fn get_output_type_string(function_type: &FunctionType) -> String {
         FunctionType::Binary(left, right, ref out_sig) => match out_sig {
             FunctionReturnsSignature::Fixed(out_type) => format!("{}", out_type),
             FunctionReturnsSignature::TypeOfArgAtPosition(pos) => {
-                let arg_sig: &FunctionArgSignature;
+                let arg_sig =
```

Comment:
Nit: the `match` statement should be indented to one scope block deeper than the `let arg_sig` lvalue.